src/Compiler/tools/Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal/Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReferemce Include="..\..\Microsoft.AspNetCore.Mvc.Razor.Extensions\src\Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj" ReferenceOutputAssembly="false" />
```

Comment:
Lol, I fixed a bunch of these when we migrated over to a single repo. They 'worked' by virtue of those projects being built already when this gets built, and the project not actually using the contents other than via content. At this point I'm not sure if it was just a bad copy-paste or something deliberate? Either way 😆